drv/gimlet-seq-api/src/lib.rs

```diff
@@ -0,0 +1,48 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! API crate for the Gimlet Sequencer server.
+
+#![no_std]
+
+use userlib::*;
+use zerocopy::AsBytes;
+
+#[derive(Copy, Clone, Debug, FromPrimitive, PartialEq)]
+pub enum SeqError {
+    IllegalTransition = 1,
+    MuxToHostCPUFailed = 2,
```

Comment:
(I don't think this error code or the next one can actually happen in practice, I'd recommend removing them unless I missed something.)

Reply:
The `IllegalTransition` can happen due to an illegal call from `hiffy`. `MuxToHostCPUFailed` *shouldn't* happen, but the task that is propagating this error (namely, the sequencer) doesn't know that:  it is propagating this on a failure from the host flash server.